### DISCUSSION TITLE
guidance scale

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2952] [Q&A] [answered]
I would like to know the guidance_scale  and how to use it.

**Comment by mashb1t** [comment, marked as answer]
You can familiarize yourself with the basics here: https://stable-diffusion-art.com/cfg-scale/

**Comment by CantainPike** [comment]
You may find some basics on YouTube before you ask, please check this channel out you will find 99 percent of your questions. Take your time there is a learning curve it takes time and continuous forever. 
https://www.youtube.com/@JumpIntoAI/videos
https://www.youtube.com/@KLEEBZTECH
https://www.youtube.com/watch?v=8krykSwOz3E&t=351s
